File: 01-Assignment-02-02-2026/Using_C/program_36.c
/* Program 36: Sum even numbers from 1 to 100 using for loop */

#include <stdio.h>

int main() {
    int sum= 0;
    for(int i = 1;i<=100;i++){
        if (i%2 == 0)
            sum+=i;
    }
    printf("Sum-> %d",sum);
    return 0;
}

File: 01-Assignment-02-02-2026/Using_Java/Program36.java
/*
Program 36
Java program to sum even numbers from 1 to 100 using for loop
*/
public class Program36 {
    public static boolean isEven(int n){
        return n%2 == 0;
    }

    public static void main(String[] args) {
        int sum = 0;
        for(int i = 1; i < 101 ; i++){
            if (isEven(i))
                sum+=i;
        }
        System.out.println("sum even numbers from 1 to 100 using for loop is " + sum);
    }

}
